public_dropin_gpu_environments/nemollm_inference_ms/env_info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "66151694d53a8a537360dbd8",
+  "name": "[Experimental][NVIDIA] NeMo Microservice Inference (24.01)",
+  "description": "NVIDIA Inference Microservice GPU accelerated LLM capabilities through OpenAI compatible API's.",
+  "programmingLanguage": "python",
+  "environmentVersionId": "66151613d53a8a4e05e903bf",
+  "isPublic": false
```

Comment:
@yakov-g, I have not found any docs / examples on how the `isPublic: false` works. Does it mean that env will be installed, but it should be somehow enabled per Org, or per user?

Reply:
https://github.com/datarobot/execution-environment-installer?tab=readme-ov-file#adding-new-environments-and-updating-existing-ones

Set it to `true` to make it public. 
Otherwise I don't have another mechanism to make it available, for example, per org.